<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>877</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="income_tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>811</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <column>
    <property name="text">
     <string>수입내역</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>금액</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>합계</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="out_tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>486</y>
     <width>811</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <column>
    <property name="text">
     <string>지출내역</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>금액</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>합계</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="income_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>55</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>익금 변동 사항</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Quarterly_income_close_Button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>42</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Quarterly_excel_save_Button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>엑셀로 저장</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bogo_year_widget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="year_label">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>9</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>년도</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Quarter_widget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>분기 특별회계 재정보고서</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Quarter_view_Button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>42</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>보기</string>
   </property>
  </widget>
  <widget class="QLabel" name="balance_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>지출 변동사항</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QPrint_Button">
   <property name="geometry">
    <rect>
     <x>627</x>
     <y>10</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>프린트</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Quarter_widget</tabstop>
  <tabstop>Quarter_view_Button</tabstop>
  <tabstop>Quarterly_excel_save_Button</tabstop>
  <tabstop>QPrint_Button</tabstop>
  <tabstop>Quarterly_income_close_Button</tabstop>
  <tabstop>bogo_year_widget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Quarter_view_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>Quarterly_view()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>QPrint_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>print_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>732</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Quarterly_excel_save_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>special_Q_excel_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Quarterly_income_close_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>special_Q_income_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Quarterly_view()</slot>
  <slot>special_Q_excel_save()</slot>
  <slot>special_Q_income_close()</slot>
  <slot>yoyak_view()</slot>
  <slot>print_button_clicked()</slot>
 </slots>
</ui>
